src/components/pickers/DatePicker.js

```diff
@@ -118,6 +139,25 @@ export default {
     cancel () {
       this.inputDate = this.originalDate
       if (this.$parent && this.$parent.isActive) this.$parent.isActive = false
+    },
+    isAllowed (date) {
+      if (!this.allowedDates)
+        return true
+
+      if (this.allowedDates instanceof Array) {
```

Comment:
This could return a false positive, as an object can be interpreted as an Array,
```
let l = []
let k = {}
l instanceof Array (true)
k instanceof Array (true)
Array.isArray(l) (true)
Array.isArray(k) (false)
````